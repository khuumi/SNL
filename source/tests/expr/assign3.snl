local var is 58.9