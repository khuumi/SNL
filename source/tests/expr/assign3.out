Assign (Id Local var) (Float 58.9)